appveyor.yml

```diff
@@ -8,11 +8,11 @@ environment:
 
   matrix:
     # XX TODO: re-enable
```

Comment:
No need of todo here as py 2.7 is enable

Reply:
But Python 3.4 is still missing :) (Even though we want to remove it eventually.)

Reply:
lets just change the position of todo for py34

Reply:
Thanks, done